streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -202,6 +206,8 @@ public void testProcessOrder() throws Exception {
         assertEquals(0, task.numBuffered());
         assertEquals(3, source1.numReceived);
         assertEquals(3, source2.numReceived);
+
+        cleanRun = true;
```

Comment:
The idea was to close the task in `@After` according to the rest result -- it the test fails, we `close(false)` -- on a clean test run we `close(true)`. I agree it's noisy, but thought it good to have. How should we call `close` in `@After` without the var? `true` or `false` ?